maven-wrapper-distribution/src/resources/only-mvnw

```diff
@@ -202,19 +202,19 @@ elif set_java_home; then
   javaSource="$TMP_DOWNLOAD_DIR/Downloader.java"
   targetZip="$TMP_DOWNLOAD_DIR/$distributionUrlName"
   cat >"$javaSource" <<-END
```

Comment:
can we use `<<END` as we don't have a tabs now?

Reply:
done and rebased.